docs/other/tips/angular.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Angular
+---
+## Tips and Tricks for Angular
+
+### Build with 0 downtime on Angular
+
+The recommended way to have 0 downtime it's using dockerized builds with a docker registry service.
+In case that this isn't possible in your project, here you have a quick fix.
+
+Create this folders on `/dist`:
+
+1. `/dist/previousbuild`
+1. `/dist/nginx` (the new production path)
+
+`package.json` add on `scripts` section:
+```json
+...
+"prebuild": "rm -rf dist/previousbuild",
+"build": "npm rebuild node-sass npm install && ng build --prod --source-map --optimization=true --buildOptimizer=true --progress=true",
```

Comment:
This looks weird: `npm rebuild node-sass npm install`, a couple of things here:
- Remove `npm rebuild node-sass`? Looks like project specific… I'm not sure why you'll need to do that
- `npm ci` instead of `npm install`? `ci` just uses what's on `package-lock.json`, while `install` takes `package.json` and resolves the dependencies which means that you might introduce a version different from the one you tested on.

Reply:
The `npm install` must be something specific for a project. Angular build is not responsible to install the NPM  dependencies and it should be something that it's expected.

Reply:
Thanks @doup and @joselufo , just updated the code with your suggestions.